eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+id: eu.usdx.UltraStarDeluxe
+branch: master
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: true
+rename-desktop-file: ultrastardx.desktop
+rename-icon: ultrastardx
+command: ultrastardx
+
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+- --filesystem=~/.java/.userPrefs/PortMidi:ro
```

Comment:
java??

Reply:
PortMidi is strange...
It's a C library that can be configured with a Java tool.

Reply:
To make sure I get it right, this is where it would be configured by some other tool?

Reply:
Yes. The Java tool saves the configuration in its [preferences](https://docs.oracle.com/en/java/javase/15/core/preferences-api1.html#GUID-2DAC3DD0-993A-41A8-8CDC-F8E3A72E1AE3__SECTION_KWW_Z1P_S3B) and the C library reads it from there.